doc/connectivity/resting_state_networks.rst

```diff
@@ -85,3 +88,60 @@ each component separately.
    displayed on your computer might not match those of the documentation. For
    a fair representation, you should display all components and
    investigate which one resemble those displayed above.
+
+Beyond ICA : Dictionary learning
+================================
+
+Recent work has shown that dictionary learning based techniques.
+Dictionary learning in neuro-imaging seek to extract a few representative
+temporal elements along with their sparse spatial loadings, which constitutes
```

Comment:
The conceptual motivation and advantages/differences over ICA and sparse PCA are not sufficiently clear, I find.